docker-compose.env

```
@@ -1,7 +1,7 @@
 AUTH0_DOMAIN="not-used"
 DOMAIN=acme.com
 LEDGER_ID="2D105384-CE61-4CCC-8E0E-37248BA935A3"
-DOCKER_IMAGE="digitalasset/daml-sdk:1.5.0"
+DOCKER_IMAGE="digitalasset/daml-sdk:1.10.1"
```

```suggestion
DOCKER_IMAGE="digitalasset/daml-sdk:1.10.0"
```
same everywhere else, 1.10.1 is not a stable sdk release, only a stable daml on sql release.

---

I will make change. I actually planned to try out this feature as part of testing against a hardened PGSQL.